readme.md

```diff
@@ -88,12 +88,12 @@ All libs listed here (bar cmake) required in 32 bit variants.
 ### Windows:
 1. Check out the repository. This can be done using [GitHub Windows](https://windows.github.com/) or [other tools](https://help.github.com/articles/which-remote-url-should-i-use).
 2. Open a new PowerShell window and navigate to the repository.
-3. Run the setenv.ps1 script in the repository to setup your PowerShell environment for OpenRCT2 development. This will warn you of any missing applications required to build OpenRCT2.
-4. Run ```install``` to download the required dependencies to build OpenRCT2.
-5. Run ```build all``` to build all the required components for OpenRCT2.
-6. Run ```run``` to run OpenRCT2.
+3. Run the ```setenv.ps1``` script in the repository to setup your PowerShell environment for OpenRCT2 development. This will warn you of any missing applications required to build OpenRCT2.
+4. Run ```./scripts/ps/install.ps1``` to download the required dependencies to build OpenRCT2.
+5. Run ```./scripts/ps/build.ps1``` to build all the required components for OpenRCT2.
+6. Run ```./scripts/ps/run.ps1``` to run OpenRCT2.
```

Comment:
I thought it might be slightly more intuitive to run the script files directly. I'll change that back.